app/util.js

```diff
@@ -296,5 +296,19 @@ module.exports = {
     } catch (err) {
         return t('notification.error.github');
     }
+  },
+
+  replaceVariables: function(variables, setting) {
+    var result = setting;
+    if (result) {
+      for (var name in variables) {
+        if (variables.hasOwnProperty(name)) {
+          var value = variables[name];
+          result = result.replace(name, value);
+        }
+      }
+    }
+    return result;
```

Comment:
This function might just be better off living in the files collection. Since it's not used anywhere else, that would help readability.

Reply:
I did try putting it in `files` initially. The problem was that the `_.each` callback functions inside `parseConfig` don't currently do `.bind(this)`, so you don't have access to the `this` object inside them. I can fix that though, so I'll do that and resubmit.